src/main/java/org/apache/commons/collections4/bidimap/AbstractDualBidiMap.java

```diff
@@ -452,7 +449,7 @@ public void clear() {
         private static final long serialVersionUID = -7107935777385040694L;
 
         /**
-         * Constructs a new view of the BidiMap.
+         * Constructs a new KeySet of the BidiMap
```

Comment:
I think the view it referred to was more in an abstract way, but I think it should be fine to be specific and mention the `KeySet` type here.